README.md

```diff
@@ -7,17 +7,119 @@ Apache SkyWalking BanyanDB Helm Chart
 BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing, and Logging data.
 It's designed to handle observability data generated by observability platforms and APM systems, like [Apache SkyWalking](https://github.com/apache/skywalking) etc.
 
-BanyanDB Helm Chart repository provides ways to install and configure SkyWalking in a BanyanDB cluster. The scripts are written in Helm 3.
+BanyanDB Helm Chart repository provides ways to install and configure BanyanDB running in a cluster natively on Kubernetes. The scripts are written in Helm 3.
```

Comment:
@hanahmily Please confirm this, I think this helm chart doesn't control skywalking OAP in any case.